src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
 
+        Machine[] machines = new Machine[] {truck, bulldozer, excavator};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+        }
+
+        for (Machine machine : machines) {
+            machine.stopWork();
+        }
```

Comment:
```suggestion
        for (Machine machine : machines) {
            machine.doWork();
            machine.stopWork();
        }
```
lets combine for loops